vernac/egramcoq.ml

```diff
@@ -268,15 +268,15 @@ let custom_entry_locality = Summary.ref ~name:"LOCAL-CUSTOM-ENTRY" String.Set.em
 let create_custom_entry ~local s =
   if List.mem s ["constr";"pattern";"ident";"global";"binder";"bigint"] then
     user_err Pp.(quote (str s) ++ str " is a reserved entry name.");
-  let sc = "constr:"^s in
+  let sc = "custom:"^s in
   let sp = "pattern:"^s in
```

Comment:
By analogy, the prefix should probably be better of the form `custom_pattern:`, or `custompattern:`, or `custom:pattern:`, since it is for patterns of a custom entry.

Reply:
With this change, this is the only point in the code that generates an entry with a prefix of `pattern:`, which is enough to assure uniqueness.  `custompattern:` seemed a little wordy.  Also, at the moment, that name is never displayed or used.  I will change it to `custom_pattern:` if you prefer.

Reply:
> Also, at the moment, that name is never displayed or used. 

Indeed, but it may be used in the future, see [#13092](https://github.com/coq/coq/pull/13092#discussion_r495894797).

Ok for `custom_pattern:` then.

Reply:
And so it shall be.  Updated.